specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-08-02-preview/SpringMiddlewares.json

```diff
@@ -0,0 +1,925 @@
+                  {
+                    "description": "Enable the eureka server.",
+                    "value": "Enabled"
+                  },
+                  {
+                    "description": "Disable the eureka server.",
+                    "value": "Disabled"
+                  }
+                ]
+              }
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ConfigServerResourceCollection": {
+      "description": "Azure Spring Apps Config Server collection ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConfigServer"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "EurekaServerResourceCollection": {
+      "description": "Azure Spring Apps Eureka Server collection ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EurekaServer"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ConfigServerSettings": {
+      "description": "The settings of config server.",
+      "type": "object",
+      "properties": {
+        "gitProperty": {
+          "$ref": "#/definitions/ConfigServerGitProperty",
+          "description": "Property of git environment."
+        }
+      }
+    },
+    "ConfigServerGitProperty": {
+      "description": "Property of git.",
+      "required": [
+        "uri"
+      ],
+      "type": "object",
+      "properties": {
+        "repositories": {
+          "description": "Repositories of git.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GitPatternRepository"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "uri": {
+          "description": "URI of the repository",
+          "type": "string"
+        },
+        "label": {
+          "description": "Label of the repository",
+          "type": "string"
+        },
+        "searchPaths": {
+          "description": "Searching path of the repository",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "username": {
+          "description": "Username of git repository basic auth.",
+          "type": "string"
+        },
+        "password": {
```

Comment:
>password [](http://example.com/codeflow?start=9&length=8)

Please use "x-ms-secret" to indicate that this is a secret: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret). Same is true for other secrets e.g. hostKey, privateKey etc. in your swagger.

Also, ensure that service implementation doesn't return the password and other secret values in PUT/GET/PATCH API responses. Secret values can only be returned using POST API response, if needed to return secret values.